app/services/idv/analytics_events_enhancer.rb

```diff
@@ -120,7 +99,7 @@ module AnalyticsEventsEnhancer
       idv_final
       idv_please_call_visited
       idv_start_over
-    ].to_set.freeze
+    ].uniq.freeze
```

Comment:
A/N: Using a `Set` causes some unexpected behaviors when doing collection comparisons as it behaves more like a `Hash` than an `Array`. Switched to using `uniq` since the const is immutable anyway. (This `uniq` call is superfluous as the list is so short, but keeping it here for consistency in behavior.)